waiter/src/waiter/handler.clj

```diff
@@ -191,11 +191,13 @@
   [request]
   (try
     (let [request-params (:params (ring-params/params-request request))
-          exclude-services (Boolean/parseBoolean (get request-params "exclude-services" "false"))
+          exclude-services (utils/request-flag request-params "exclude-services")
           service-id (get request-params "service-id" nil)
+          show-jvm-metrics (utils/request-flag request-params "show-jvm-metrics")
```

Comment:
Should this be named `include-jvm-metrics` to be consistent with `exclude-services`?